.github/test-dispatch.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
```

Comment:
Does this script work now?

Reply:
Yes, it works at the moment, but It's just a testing script. We really could just delete this file before merging.